tests/Action/SetObjectFieldValueActionTest.php

```diff
@@ -344,14 +367,130 @@ public function testSetObjectFieldEditableMultipleValue(): void
         $fieldDescription->getOption('editable')->willReturn(true);
         $fieldDescription->getOption('multiple')->willReturn(true);
         $fieldDescription->getAdmin()->willReturn($this->admin->reveal());
-        $fieldDescription->getType()->willReturn('boolean');
-        $fieldDescription->getTemplate()->willReturn('field_template');
+        $fieldDescription->getType()->willReturn(null);
+        $fieldDescription->getTemplate()->willReturn(false);
```

Comment:
Could we return `null` instead?

Reply:
I restore origin value but yes, `null` can be returned in `getTemplate()` method as far as i can tell.